src/Go/ui/GoGameboardWrapper.tsx

```diff
@@ -119,9 +117,9 @@ export function GoGameboardWrapper({ showInstructions }: GoGameboardWrapperProps
   }
 
   async function takeAiTurn(boardState: BoardState) {
-    // If white is being played manually, halt and notify any scripts playing as black if present, instead of making an AI move
+    // Instead of making an AI move on the "No AI" opponent, notify any scripts waiting for their turn and then halt
```

Comment:
This code shouldn't be needed; the script-play version doesn't have it, but relies on `makeAIMove` to do the right thing. It's better if they both go through the same codepath.